docs/deployment/security/sbom.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Software Bill of Materials (SBOM)'
+description: 'Generating SBOMs with wasmCloud'
+sidebar_position: 5
+---
+# Securing your wasmCloud supply chain with SBOMs
+
+[**Software Bills of Materials (SBOMs)**](https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) describe the set of software dependencies (sometimes called the "software supply chain") used to build an application. 
```

Comment:
I realized after I wrote up my notes that I should've really referred to the official source for this
```suggestion
[**Software Bills of Materials (SBOMs)**](https://www.cisa.gov/sbom) describe the set of software dependencies (sometimes called the "software supply chain") used to build an application. 
```